server/serviceHelpers.js

```diff
@@ -0,0 +1,54 @@
+var axios = require('axios');
+
+// service ports TODO: move to config file
+const servicePorts = {
+  REVIEWS: '3004',
+  PHOTOS: '3003'
+};
+
+// get data from reviews-service and trail-photos-service for the Banner
+// component endpoint
+var getBannerData = (id, callback) => {
+  var result = {};
+  var trailStats = `http://localhost:${servicePorts.REVIEWS}/${id}/trailStats`;
+  var trailRank = `http://localhost:${servicePorts.REVIEWS}/${id}/trailRank`;
+  var heroPhoto = `http://localhost:${servicePorts.PHOTOS}/${id}/heroPhoto`;
+  var photosCount = `http://localhost:${servicePorts.PHOTOS}/${id}/photosCount`;
+
+  axios.get(trailStats)
+    .then((response) => {
+      result.stats = response.data;
+    })
+    .catch((error) => {
+      console.error('axios.get trailStats: ', error);
+    });
+
+  axios.get(trailRank)
```

Comment:
are each of these async? consider either using async / await or a Promise.all to ensure you get all data before resolving this promise / running your callback. @MengRS  @McCloudFace